templates/docred/templates.yaml

```diff
@@ -76,6 +98,41 @@ templates:
     name: ner-to-text
     reference: ''
     task_template: false
+  7f6bb96c-3661-4369-8d75-6eca07f15e6d: !Template
+    id: 7f6bb96c-3661-4369-8d75-6eca07f15e6d
+    jinja: "{%- set time = [] -%} \n{%- for ners in vertexSet -%}\n{%- for ner in\
+      \ ners if ner['type'] == 'TIME' -%}\n{{time.append(ner['name']) | default(\"\
+      \", True)}}\n{%- endfor -%}\n{%- endfor -%}\n{% if time %}\nFind all of the\
+      \ time in the text below. \n\n{% for sent in sents -%}\n{{ sent | join(\" \"\
+      ) }}\n{%- endfor -%} \n|||\n{{ time| unique | join(\", \")}}{{\".\"}}\n{% endif\
+      \ %}"
+    name: find-all-time
+    reference: ''
+    task_template: false
+  901e34b8-70c9-4434-944e-03e38286a1c8: !Template
+    id: 901e34b8-70c9-4434-944e-03e38286a1c8
+    jinja: "{%- set misc = [] -%} \n{%- for ners in vertexSet -%}\n{%- for ner in\
```

Comment:
It might be best to remove this one. I'm a bit confused about what is considered or not considered an entity for the purposes of "MISC". For example:

Washington Place ( William Washington House ) is one of the first homes built by freed slaves after the Emancipation Proclamation of 1863 in Hampshire County , West Virginia , United States .Washington Place was built by William and Annie Washington in north Romney between 1863 and 1874 on land given to Annie by her former owner , Susan Blue Parsons of Wappocomo plantation .William Washington later acquired other properties on the hills north of Romney along West Virginia Route 28 and became the first African - American land developer in the state of West Virginia .One of his subdivisions is the " Blacks Hill " neighborhood of Romney , adjacent to the Washington Place homestead .Washington Place was bought and restored by Ralph W. Haines , a local attorney and historic preservationist .

Y

Emancipation Proclamation, Blacks Hill.

Why is "Blacks Hill" a MISC and not a location? Why isn't William Washington House a MISC? or Wappocomo? WDYT?

Reply:
yeah I thought the same. agreed!